src/components/EditReport.jsx

```diff
@@ -27,7 +28,7 @@ const EditReport = () => {
       setOpenReport(null);
       setRerenderReports(true);
     } catch (error) {
-      error instanceof Response ? webError(error) : dispatch(NotificationActions.error(t('error.exception')));
+      exception(error);
```

Comment:
in the future, we could prepare another api wrapper and avoid making changes in every api call

Reply:
It's a good thing for the future.